docs/api/index.js

```diff
@@ -0,0 +1,61 @@
+const _ = require('lodash')
+const fsPromises = require('fs').promises
+const path = require('path')
+
+const docsLib = require('docs/api/lib')
+
+const docsConfiguration = {
+  // location of the routes
+  routeDir: 'route',
+  // location of the route extended docs
+  docsRouteDir: 'route',
+  /**
+   * Automatic docs will add these files and combine with the extended docs
+   * If you want to use subdirs in the "route" folder then add the paths as arrays
+   * e.g. "route/user/friends"
+   *
+   * routeFilePaths: [ ['user', 'friends'] ]
+   */
+  routeFilePaths: [
+    'index',
+    'user',
+  ],
+}
+
+// automatic router docs
+function requireRoutes (docsConfig) {
+  return _.map(docsConfig.routeFilePaths, routeFileName => {
+    const routeFilePath = path.join(docsConfig.routeDir, ..._.castArray(routeFileName))
+    return require(routeFilePath).routes()
+  })
```

Comment:
Paths in `require` have always "/" for the separator (not OS specific), so there's no point in using `path.join` here.

```suggestion
  return _.map(docsConfig.routeFilePaths, routeFilePath => {
    return require(`${docsConfig.routeDir}/${routeFilePath}`).routes()
  })
```